sky/task.py

```diff
@@ -194,6 +195,9 @@ def __init__(
         self.estimated_outputs_size_gigabytes = None
         # Default to CPUNode
         self.resources = {sky.Resources()}
+        # Resources that this task cannot run on.
+        self.blocked_resources = blocked_resources
```

Comment:
Consider adding a comment to explain the purpose of the `blocked_resources` attribute in the `Task` class. This will help developers understand its role and how it is used. [medium]